src/app/GenericReadHandlerCallback.h

```diff
@@ -0,0 +1,47 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include "ReadHandler.h"
+
+namespace chip {
+namespace app {
+
+class GenericReadHandlerCallback : public ReadHandler::ApplicationCallback
+{
+    CHIP_ERROR OnSubscriptionRequested(ReadHandler & aReadHandler, Transport::SecureSession & aSecureSession) override
+    {
+        uint16_t interval_u16 = 0;
+        uint32_t interval_u32 = CHIP_DEVICE_CONFIG_SED_IDLE_INTERVAL.count() / 1000;
```

Comment:
This policy is very simplistic. I would rather recommend that we allow clients to set a version themselves rather than having this default always be the one used. Many products could be stateful in the sub intervals they extend based on current battery life, or based on different parameters than the SED interval. It would be best if the policy was set always by clients, rather than via Server init.

Reply:
done